### PR TITLE
chore(package.json): update packages / resolve dependency issues

### PR DESCRIPTION
**Pull request by johnleider**
## Description
Update all dependencies to latest version

## Motivation and Context
Resolve dependency errors and stay up to date

## How Has This Been Tested?
jest / visually

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

**Comment by johnleider** [comment]
Requesting **Final** comments from @vuetifyjs/contributors @vuetifyjs/core-team

File: packages/docs/package.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
     "webfontloader": "^1.6.28"
   },
   "devDependencies": {
-    "@babel/core": "^7.5.5",
+    "@babel/core": "^7.0.0-0",
```
Comment (sh7dm) [inline packages/docs/package.json:53]: Why downgrading here?

File: packages/vuetify/package.json [hunk 2]
```diff
@@ -65,7 +65,7 @@
   },
   "devDependencies": {
     "@babel/cli": "^7.5.5",
-    "@babel/core": "^7.5.5",
+    "@babel/core": "^7.0.0-0",
```
Comment (sh7dm) [inline packages/vuetify/package.json:68]: Unsure about this downgrade too

File: packages/docs/package.json (hunk 1 above)
Comment (johnleider) [reply packages/docs/package.json:53]: I'll double check the warnings associated.

File: packages/vuetify/package.json (hunk 2 above)
Comment (johnleider) [reply packages/vuetify/package.json:68]: I'll double check the warnings associated.

**Comment by phiter** [comment]
I see we're still using `@vue/test-utils: 1.0.0-beta.29`.

I upgraded my company's project to the latest version (`1.0.0-beta.33`).

I had to change a lot of stuff because they removed the `sync` behavior which some of our tests relied on. The change is actually a good thing, but we'll have to keep that in mind when updating Vuetify as well.

**Pull request closed without merging**